### DISCUSSION TITLE
"Announce" vs "Announce-list" difference from Transmission 3 > Transmission 4?

### DISCUSSION THREAD

**Discussion by manofsticks** [transmission/transmission#5103] [Support] [answered]
In a private tracker I'm apart of, torrent files are not uploading properly when created from Transmission 4.0.1.

Talking with forum users, one made the following comment:

> We had that problem on other site and we figured out that when creating .torrent with transmission 4.x it puts announce url into "announce-list" when all other clients (also transmission 3.x) puts it into "announce", so it seems that its just a new bug.

Was curious if I could get more information on this, and if it's truly a "bug" and I should make an issue (did not find an existing one), or if it's a feature change that the tracker should be expected to handle. 

Thanks

**Comment by ckerr** [comment, marked as answer]
Moved to an issue, https://github.com/transmission/transmission/issues/5105
